dags/aps/APSApiClient.py

```diff
@@ -0,0 +1,30 @@
+from os import getenv
+import json
+
+from common.request import Request
+
+class APSApiClient(Request):
```

Comment:
The class looks great to me ! I would just change the relationship between the class and the `Request` one : The API client is not a Request, it uses requests. Let's take "get_articles_metadata" for example : 

```python
def get_articles_metadata(self):
  request = Request(self.base_url, self.port, parameters, headers)
  response_content = request.get_response_content()
  ...
```